docs/reference/commands.md

```diff
@@ -0,0 +1,104 @@
+---
+id: commands
+title: Command reference
+description: A Command is a requested action issued by a Worker to the Temporal Cluster, after the Worker has finished completing the execution of a Workflow Task.
+tags:
+  - reference
+  - explanation
+---
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowTask from '../content/what-is-a-workflow-task.md'
+import * as WhatIsAWorker from '../content/what-is-a-worker.md'
+import * as WhatIsATemporalCluster from '../content/what-is-a-temporal-cluster.md'
+import * as WhatIsAWorkflowExecution from '../content/what-is-a-workflow-execution.md'
+import * as WhatIsAnEventHistory from '../content/what-is-an-event-history.md'
+import * as WhatIsAnEvent from '../content/what-is-an-event.md'
+
+### What is a Command?
+
+A Command is a requested action issued by a <preview page={WhatIsAWorker}>Worker</preview> to the <preview page={WhatIsATemporalCluster}>Temporal Cluster</preview> after the Worker has finished completing the execution of a <preview page={WhatIsAWorkflowTask}>Workflow Task</preview>.
+
+The action that the Cluster takes is recorded in the <preview page={WhatIsAWorkflowExecution}>Workflow Execution's</preview> <preview page={WhatIsAnEventHistory}>Event History</preview> as an <preview page={WhatIsAnEvent}>Event</preview>.
+
+During the execution of a Workflow Task there may be several Commands that are generated.
+The Commands are batched and sent to the Cluster as part of the Workflow Task completion request, after the Workflow Task has progressed as far as it can.
+
+### CompleteWorkflowExecution
+
+- Description: This Command is triggered when the last Workflow Task of the Workflow Execution has finished executing successfully.
+  Often this is when the last line of code in a Workflow Definition has been execution.
```

Comment:
"when the last Workflow Task of the Workflow Execution has finished executing successfully." is confusing.

This command indicates to the service that workflow execution is completed. This is always the last command in the workflow execution history. 

We can also mention that this command contains the result of a workflow execution.